core/editor.js

```diff
@@ -1430,28 +1430,36 @@
 		 *
 		 * @since 4.6.0
 		 * @param {CKEDITOR.command/String} command The {@link CKEDITOR.command} instance or a string with the command name.
-		 * @returns {Number/null} The keystroke assigned to the provided command or `null` if there is no keystroke.
+		 * @param {Boolean} [all=false] If `true` function will return an array of the assigned keystrokes.
+		 * Available since 4.11.0.
+		 * @returns {Number/Number[]/null} Depending on the `all` parameter value:
+		 *
+		 * `false` (default): the first keystroke assigned to the provided command or `null` if there is no keystroke.
```

Comment:
This proposal looks really nice. I'll just remove info about `false` being a default value of `all` attribute as it is indicated by `[all=false]` and it should be the only source of truth for this information.